source/refined-github.css

```diff
@@ -116,3 +116,8 @@ pr-branches
 .js-issue-row .text-small.text-gray {
 	line-height: 1.8;
 }
+
+/* Reset `vertical-align` for .octicons to work with above increased line-height */
+.js-issue-row .text-small.text-gray .octicon {
```

Comment:
```suggestion
.rgh-align-issue-labels .js-issue-row .text-small.text-gray .octicon {
```
With `rgh-align-issue-labels` turned off the alignment is a drop off

Reply:
This is unrelated to that feature, the spacing changes regardless of that

Reply:
@yakov116 The highlighted change is concerned with the alignment of the `clock` icon which looks correct with `align-labels` turned off:

![Screenshot from 2020-05-28 13-51-21](https://user-images.githubusercontent.com/1682504/83138004-aaac7e00-a0d9-11ea-8bc6-b7ee3c99d4f2.png)

Reply:
Your both right, I thought i noticed  a small change.